src/components/Loader/PersonInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import { NavLink, useLocation } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type PersonInfoProps = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonInfo = ({ person, people }: PersonInfoProps) => {
```

Comment:
`./Loader` directory should not contain `PersonInfo` component. It is counterintuitive